docs/user-guide/overview.md

```diff
@@ -31,6 +31,10 @@ To use GreptimeDB as a log storage solution,
 refer to the [Logs Documentation](./logs/overview.md).
 It details how to ingest pattern text logs using pipelines.
 
+To use GreptimeDB as a trace storage solution,
```

Comment:
What about using a list form:

* For logs user....
* For traces users ...

Reply:
According to the developer documentation style guide:
> address the reader of your documents using the second person instead of the first person
> Use the word `user` only to refer to the user of the software that your reader is developing.

If you want to let the `log/trace` appear at the beginning of the sentences, we can change it to:
- For log storage solution...
- For trace storage solution...

Reply:
I meant it to be a markdown list, but that's ok.